pom.xml

```diff
@@ -76,7 +76,7 @@
         <assertj.version>3.16.1</assertj.version>
         <junit.version>5.7.0</junit.version>
         <org.apache.commons.io.version>2.7</org.apache.commons.io.version>
-        <com.fasterxml.jackson.version>2.13.2</com.fasterxml.jackson.version>
+        <com.fasterxml.jackson.version>2.15.2</com.fasterxml.jackson.version>
         <com.fasterxml.jackson-databind.version>2.13.4.2</com.fasterxml.jackson-databind.version>
```

Comment:
Can we step jackson-databind as well? It feels better if databind and other jackson libs follow the same version.

Reply:
Good point. I'll step that as well.